### ISSUE TITLE
Project will not build when upgrading from 3.6.0 to 3.6.2 

### ISSUE THREAD

**Issue by pmlopes** [vert-x3/vertx-guide-for-java-devs#59]
```
cd step-10
# edit bom to be 3.6.2 in place of 3.6.0
mvn compile
```

Output:

```
[INFO] Scanning for projects...
[INFO]
[INFO] -----------------------< io.vertx:wiki-step-10 >------------------------
[INFO] Building wiki-step-10 1.5.0-SNAPSHOT
[INFO] --------------------------------[ jar ]---------------------------------
[INFO]
[INFO] --- maven-clean-plugin:3.0.0:clean (default-clean) @ wiki-step-10 ---
[INFO] Deleting /home/plopes/Downloads/vertx-guide-for-java-devs-3.6/step-10/target
[INFO] Deleting /home/plopes/Downloads/vertx-guide-for-java-devs-3.6/step-10/src/main/generated (includes = [], excludes = [])
[INFO]
[INFO] --- maven-resources-plugin:2.6:resources (default-resources) @ wiki-step-10 ---
[INFO] Using 'UTF-8' encoding to copy filtered resources.
[INFO] Copying 4 resources
[INFO]
[INFO] --- maven-compiler-plugin:3.5.1:compile (default-compile) @ wiki-step-10 ---
[INFO] Compiling 8 source files to /home/plopes/Downloads/vertx-guide-for-java-devs-3.6/step-10/target/classes
Jan 29, 2019 12:22:12 PM io.vertx.codegen.CodeGenProcessor getCodeGenerators
WARNING: Please use 'codegen.output' option instead of 'outputDirectory' option
Jan 29, 2019 12:22:12 PM io.vertx.codegen.CodeGenProcessor lambda$getCodeGenerators$4
INFO: Loaded service_proxy_handler code generator
Jan 29, 2019 12:22:12 PM io.vertx.codegen.CodeGenProcessor lambda$getCodeGenerators$4
INFO: Loaded service_proxy code generator
Jan 29, 2019 12:22:12 PM io.vertx.codegen.CodeGenProcessor lambda$getCodeGenerators$4
INFO: Loaded cheatsheet code generator
Jan 29, 2019 12:22:12 PM io.vertx.codegen.CodeGenProcessor lambda$getCodeGenerators$4
INFO: Loaded cheatsheet code generator
Jan 29, 2019 12:22:12 PM io.vertx.codegen.CodeGenProcessor lambda$getCodeGenerators$4
INFO: Loaded data_object_converters code generator
Jan 29, 2019 12:22:13 PM io.vertx.codegen.CodeGenProcessor lambda$process$8
INFO: Generated model io.vertx.guides.wiki.database.WikiDatabaseService: io.vertx.guides.wiki.database.WikiDatabaseServiceVertxProxyHandler
Jan 29, 2019 12:22:13 PM io.vertx.codegen.CodeGenProcessor lambda$process$8
INFO: Generated model io.vertx.guides.wiki.database.WikiDatabaseService: io.vertx.guides.wiki.database.WikiDatabaseServiceVertxEBProxy
[INFO] -------------------------------------------------------------
[ERROR] COMPILATION ERROR :
[INFO] -------------------------------------------------------------
[ERROR] /home/plopes/Downloads/vertx-guide-for-java-devs-3.6/step-10/src/main/java/io/vertx/guides/wiki/database/WikiDatabaseService.java:[47,49] package io.vertx.guides.wiki.database.reactivex does not exist
[ERROR] /home/plopes/Downloads/vertx-guide-for-java-devs-3.6/step-10/src/main/java/io/vertx/guides/wiki/http/HttpServerVerticle.java:[27,47] package io.vertx.guides.wiki.database.reactivex does not exist
[ERROR] /home/plopes/Downloads/vertx-guide-for-java-devs-3.6/step-10/src/main/java/io/vertx/guides/wiki/http/HttpServerVerticle.java:[53,11] cannot find symbol
  symbol:   class WikiDatabaseService
  location: class io.vertx.guides.wiki.http.HttpServerVerticle
[INFO] 3 errors
[INFO] -------------------------------------------------------------
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 2.417 s
[INFO] Finished at: 2019-01-29T12:22:13+01:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.5.1:compile (default-compile) on project wiki-step-10: Compilation failure: Compilation failure:
[ERROR] /home/plopes/Downloads/vertx-guide-for-java-devs-3.6/step-10/src/main/java/io/vertx/guides/wiki/database/WikiDatabaseService.java:[47,49] package io.vertx.guides.wiki.database.reactivex does not exist
[ERROR] /home/plopes/Downloads/vertx-guide-for-java-devs-3.6/step-10/src/main/java/io/vertx/guides/wiki/http/HttpServerVerticle.java:[27,47] package io.vertx.guides.wiki.database.reactivex does not exist
[ERROR] /home/plopes/Downloads/vertx-guide-for-java-devs-3.6/step-10/src/main/java/io/vertx/guides/wiki/http/HttpServerVerticle.java:[53,11] cannot find symbol
[ERROR]   symbol:   class WikiDatabaseService
[ERROR]   location: class io.vertx.guides.wiki.http.HttpServerVerticle
[ERROR] -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
```

**Comment by jponge** [comment]
The master branch was already ok.

**Comment by jponge** [comment]
Fixed by f49032d66856a6945477240a3bf559fcc79525e0
